README.md

```diff
@@ -11,20 +11,19 @@ These core user-facing software include:
 - [JupyterHub](https://github.com/jupyterhub/jupyterhub)
 
 > **Important notice**
-> As of March 2019 this software **is not accessible.**
-> Significant work will be required to reach that goal.
+> As of March 2022 this software **is not accessible.**
+> Significant work will be required to reach that goal. In the meantime, you can track the progress and initiatives in our [Jupyter Accessibility Team Compass](https://jupyter-accessibility.readthedocs.io)
 
-> **Note**
-> As from June 2022 this project is an official Software Subproject as per [Jupyter Governance](https://jupyter.org/governance/software_subprojects.html)
+## Using Jupyter tools with assistive technology
 
 If you are trying to use Jupyter tools with assistive technology now, you may be able to get basic support with the following:
 
-- [Jupyter Notebook a11y toolbar](https://github.com/uclixnjupyternbaccessibility/jupyter_contrib_nbextensions/tree/master/src/jupyter_contrib_nbextensions/nbextensions/accessibility_toolbar) (from Microsoft research)
+- [Jupyter Notebook a11y toolbar](https://github.com/uclixnjupyternbaccessibility/jupyter_contrib_nbextensions/tree/master/src/jupyter_contrib_nbextensions/nbextensions/accessibility_toolbar) (from Microsoft research - ⚠️ note this has not been maintained since 2019)
```

Comment:
should we just not recommend it then?

Reply:
🤷🏻‍♀️ I am tempted to say no - but AFAIK this is still the only extension of its type. I added the warning as others might feel differently about removing this.

cc @isabela-pf

Reply:
I think it may be worth keeping in case people are working with something compatible with this tool. In my experience, lots of people are not using the most recent Jupyters and this could be a big help.